tasks.rb

```diff
@@ -172,6 +172,20 @@ def create_osws
     'base-hvac-room-ac-only.osw' => 'base.osw',
     'base-hvac-room-ac-only-33percent.osw' => 'base.osw',
     'base-hvac-setpoints.osw' => 'base.osw',
+    # 'base-hvac-shared-boiler-chiller-fan-coil.osw' => 'base.osw',
+    # 'base-hvac-shared-boiler-chiller-fan-coil-ducted.osw' => 'base.osw',
+    # 'base-hvac-shared-boiler-chiller-water-loop-heat-pump.osw' => 'base.osw',
+    # 'base-hvac-shared-boiler-cooling-tower-water-loop-heat-pump.osw' => 'base.osw',
+    # 'base-hvac-shared-boiler-only-baseboard.osw' => 'base.osw',
+    # 'base-hvac-shared-boiler-only-fan-coil.osw' => 'base.osw',
+    # 'base-hvac-shared-boiler-only-fan-coil-ducted.osw' => 'base.osw',
+    # 'base-hvac-shared-boiler-only-water-loop-heat-pump.osw' => 'base.osw',
+    # 'base-hvac-shared-chiller-only-baseboard.osw' => 'base.osw',
+    # 'base-hvac-shared-chiller-only-fan-coil.osw' => 'base.osw',
+    # 'base-hvac-shared-chiller-only-fan-coil-ducted.osw' => 'base.osw',
+    # 'base-hvac-shared-chiller-only-water-loop-heat-pump.osw' => 'base.osw',
+    # 'base-hvac-shared-cooling-tower-only-water-loop-heat-pump.osw' => 'base.osw',
+    # 'base-hvac-shared-ground-loop-ground-to-air-heat-pump.osw' => 'base.osw',
```

Comment:
Why are these all commented out? With 'extra-' versions used instead (below)?

Reply:
Because the geometry portions of the built hpxml files have discrepancies with the test xmls...